tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import pytest
+
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.framework.pytest_customization.marks import (
+    magenta_squad,
+    skipif_ocs_version,
+    skipif_ocp_version,
+    bugzilla,
+)
+from ocs_ci.framework.testlib import E2ETest, tier2
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import cluster
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources.pod import get_pod_logs
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def run_metadata_io_with_cephfs(dc_pod_factory):
```

Comment:
is this the same code that we are using for mds mem and cpu alert feature? If yes, should we move it some common place and call that function in the test?